do/serverless.go

```diff
@@ -276,7 +303,7 @@ type ServerlessOutput struct {
 }
 
 // NewServerlessService returns a configured ServerlessService.
-func NewServerlessService(client *godo.Client, usualServerlessDir string, credsToken string) ServerlessService {
+func NewServerlessService(client *godo.Client, usualServerlessDir string, accessToken string) ServerlessService {
```

Comment:
We need the actual access token, not just a hash of it, to support invoking the deployer and enabling it to manage triggers.